### DISCUSSION TITLE
More Settings for Windows 10 Start Menu

### DISCUSSION THREAD

**Discussion by MysterionTheUmbreon** [valinet/ExplorerPatcher#3346] [Ideas]
Hi, I am new here. I have some (probably not good) ideas for new settings for the Windows 10 Start Menu if they're possible:
1. Colored Tiles - This will let users have the option to re-enable the colored tiles without using regedit. (This should also work for Windows 11 users that are using the Windows 10 start menu)
2. Custom Start Menu Background - This one is just something I randomly came up with, this setting will let users have the option to choose an image for their start menu instead of the plain color or transparency.
3. Horizontal Scroll (Full screen start only) - For those who are using the windows 10 start menu on full screen, this option will let users choose from the default vertical scroll (up/down) or horizontal scroll (left/right). 
I hope at least one of these ideas are possible. If these aren't possible, please disregard this, again I am new. Thank you for your time.

**Comment by pyrates999** [comment]
Windows 10 start menu is gone in 24h2.  So don't expect any of these to be implemented.

**Reply by mdio33** [reply]
> Windows 10 start menu is gone in 24h2. So don't expect any of these to be implemented.

Is that final or there is hope that we could still use Windows 10 Start Menu in future versions?

**Reply by pyrates999** [reply]
Not if the code has been removed.  EP can only enable features where the code exists.

**Reply by mdio33** [reply]
> Not if the code has been removed. EP can only enable features where the code exists.

I understand... I hate de W11 Start Menu, so far i been happily using EP, but i guess this is the end. :(

**Reply by milestprower92** [reply]
> Windows 10 start menu is gone in 24h2. So don't expect any of these to be implemented.

you can still restore and use the windows 10 SM on WIndows 11 24H2+ if you copy 22621's startui.dll to the startmenuexperiencehost folder.
but you will need to copy the .dll after a cumulative update

**Reply by pyrates999** [reply]
Yes, and you will have to do that every time after a windows update is installed.  If it was updated because of a security vulnerability, you're opening yourself up to that.

**Reply by Zeratoxx** [reply]
The lack of features with the W11 taskbar is hindering my productiveness.. why I use the W10 taskbar..
and the Windhawk project is.. too little yet I would say

**Reply by mdio33** [reply]
> > Windows 10 start menu is gone in 24h2. So don't expect any of these to be implemented.
> 
> you can still restore and use the windows 10 SM on WIndows 11 24H2+ if you copy 22621's startui.dll to the startmenuexperiencehost folder. but you will need to copy the .dll after a cumulative update

Hello. 
That stop working on 24H2 build 26120.1330 :(

**Reply by milestprower92** [reply]
> > > Windows 10 start menu is gone in 24h2. So don't expect any of these to be implemented.
> > 
> > you can still restore and use the windows 10 SM on WIndows 11 24H2+ if you copy 22621's startui.dll to the startmenuexperiencehost folder. but you will need to copy the .dll after a cumulative update
> 
> Hello. 
> That stop working on 24H2 build 26120.1330 :(

The 10 start menu code was completely removed on those builds

**Reply by mdio33** [reply]
So... this is the end for Windows 10 Start Menu?

**Reply by Amrsatrio** [reply]
I have a trick up my sleeves for this. Rest assured. The 10 start menu is here to stay.
